db/schema.rb

```diff
@@ -15,7 +15,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "pg_stat_statements"
 
```

Comment:
@schneems I just realised that this might be added from Heroku dump and we need it for profiling. Should we add a migration to create this it it does not exist? So that running `rake db:migrate` locally will not show changes.